.storybook/themes/news.scss

```diff
@@ -210,6 +220,81 @@
 				right-rail-advanced-main-right-rail-bottom: (
 					grid-area: main-right-rail-bottom,
 				),
+				single-chain: (
+					components: (
+						stack: (
+							gap: var(--global-spacing-5),
+						),
+						heading: (
+							font-size: var(--global-font-size-10),
+							line-height: var(--global-line-height-5),
+							font-weight: var(--global-font-weight-7),
+						),
+					),
+				),
+				single-chain-children-stack: (
+					gap: var(--global-spacing-5),
+				),
+				double-chain: (
+					components: (
+						stack: (
+							gap: var(--global-spacing-5),
+						),
+						heading: (
+							font-size: var(--global-font-size-10),
+							line-height: var(--global-line-height-5),
+							font-weight: var(--global-font-weight-7),
+						),
+					),
+				),
+				double-chain-children-grid: (
+					grid-template-columns: 1fr,
+					grid-column-gap: 0,
+					grid-row-gap: var(--global-spacing-5),
```

Comment:
You can use the short hand `gap` property to apply the same logic

```suggestion
					gap: var(--global-spacing-5),
```

Reply:
It's different though, the original sets a 0 gap for columns and a `spacing-5` gap for rows.

Reply:
technically, the original is `gap: var(--global-spacing-5) 0,`

Reply:
I'd like to understand why it is not working - `gap` should work and I'd rather use that to ensure we are consistent. I tested locally and it is working for me as expected

Reply:
@visionbegin - I made a separate PR off of this one to test the `gap` change, Chromatic is showing no changes between this PR, can you confirm if you you are seeing the issue you mentioned previous

https://github.com/WPMedia/arc-themes-blocks/pull/1350